From 2f9a1d413c983a7faee04db2581d6cb81de2bf14 Mon Sep 17 00:00:00 2001
From: xzl <xiangzelong@deepin.org>
Date: Tue, 30 Jan 2024 14:37:35 +0800
Subject: [PATCH 1/1] add-desktop-file-NoDisplay=true

---
 im-config.desktop.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/im-config.desktop.in b/im-config.desktop.in
index b564a6c..1af0521 100644
--- a/im-config.desktop.in
+++ b/im-config.desktop.in
@@ -8,3 +8,4 @@ Terminal=false
 Icon=input-keyboard
 Categories=Settings
 X-AppStream-Ignore=true
+NoDisplay=true
-- 
2.43.0

